### PR TITLE
Fix native attachment preview not rendering

### PR DESCRIPTION
**Pull request by buoyad**
Also fixes max width for attachments and attachment upload error handling. cc @mmaxim r? @keybase/react-hackers

### CODE REVIEW

File: shared/actions/chat2/index.js [hunk 1]
```diff
@@ -1599,9 +1599,10 @@ function* attachmentUpload(action: Chat2Gen.AttachmentUploadPayload) {
     if (ordinal) {
       yield Saga.put(Chat2Gen.createAttachmentUploaded({conversationIDKey, ordinal}))
     }
-  } catch (e) {}
-  // TODO better error
-  logger.warn('Upload Attachment Failed')
+  } catch (e) {
+    // TODO better error
+    logger.warn(`Upload Attachment Failed: ${e}`)
```
Comment (jzila) [inline shared/actions/chat2/index.js]: debugging left in on purpose?

Comment (buoyad) [reply shared/actions/chat2/index.js]: An earlier PR moved the `logger.warn` out of the `catch`, so it was getting logged on every run.

**Comment by mmaxim** [comment]
Does this make it render not right on the right side of the screen?

**Comment by buoyad** [comment]
@mmaxim Yup

**Comment by buoyad** [comment]
I think the larger avatars broke that originally

**Pull request merged**